packages/atomic/src/components/common/generated-answer/atomic-citation/atomic-citation.tsx

```diff
@@ -102,6 +102,24 @@ export class AtomicCitation {
     );
   }
 
+  private anchorUrl(
+    uri: string,
+    text?: string,
+    filetype?: string, 
+    pageNumber?: number
+  ) {
+    if (this.disableCitationAnchoring || !text) {
+      return uri;
+    }
+    if (filetype === 'html') {
+      return this.generateTextFragmentUrl(uri, text, filetype);
+    } 
+    if (filetype === 'pdf') {
+      return this.generatePdfPageUrl(uri, pageNumber);
+    }
+    return uri;
```

Comment:
I think a `switch` would be more idiomatic and also mark a distinction between the guard (L111-113) and the logic piping/meat of the function.
```suggestion
    switch(fileType) {
      case 'html':
        return this.generateTextFragmentUrl(uri, text, filetype);
      case 'pdf':
        return this.generatePdfPageUrl(uri, pageNumber);
      default:
        return uri;
    }
```